torch/_dynamo/output_graph.py

```diff
@@ -929,6 +929,14 @@ def append_prefix_insts():
                 graph_output_var,
                 tempvars={val: None for val, count in pass1.uses.items() if count > 1},
             )
+
+            if torch._dynamo.config.reorder_prints and len(tx.debug_locals) != 0:
+                for debug_var, args in tx.debug_locals:
+                    pass2(debug_var)
```

Comment:
I think you may need to do this stuff in pass1 as well if any temporary variables need to get constructed.